docs/complexity.md

```diff
@@ -51,6 +51,15 @@ In next step, you need to integrate `graphql-query-complexity` with your graphql
         onComplete: (complexity: number) => {
           console.log("Query Complexity:", complexity);
         },
+        estimators: [
+          fieldConfigEstimator(),
+          // Add more estimators here...
```

Comment:
It's not a good suggestion - which one, directives?

Reply:
removed